index-handlers/src/main/java/no/sikt/nva/nvi/index/aws/CandidateQuery.java

```diff
@@ -161,7 +165,11 @@ private static Query yearQuery(String year) {
 
     }
 
+    private static Query categoryQuery(String category) {
+        return termQuery(Optional.ofNullable(category).orElse(EMPTY_STRING),
+                         jsonPathOf(PUBLICATION_DETAILS, TYPE, KEYWORD));
```

Comment:
The Optional should be "outside" the termQuery() instead. No need to create a termQuery for ""

Reply:
Done.